storage/plugins/book-club/src/QuoteController.php

```
@@ -0,0 +1,443 @@
+     * @param array<string, mixed> $club
+     * @return list<array<string, mixed>>
+     */
+    private function selectableBooks(array $club): array
+    {
+        $books = $this->repo->clubBooks((int) $club['id']);
+        if ($this->canManage($club)) {
+            return $books;
+        }
+        return array_values(array_filter(
+            $books,
+            static fn(array $b): bool => (string) $b['state'] !== BookClubPlugin::STATE_PENDING
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $b. Configured minimum length is 3. (undefined)

(ShortVariable)